packages/patternfly-4/react-core/src/components/Pagination/OptionsToggle.tsx

```diff
@@ -41,16 +43,17 @@ export const OptionsToggle: React.FunctionComponent<OptionsToggleProps> = ({
   showToggle = true,
   onToggle = (_isOpen: boolean) => undefined as any,
   isOpen = false,
+  isDisabled = false,
   parentRef = null,
   toggleTemplate: ToggleTemplate = '',
 }:OptionsToggleProps ) => {
   return (
-    <div className={css(styles.optionsMenuToggle, getModifier(styles, 'plain'), getModifier(styles, 'text'))} >
+    <div className={css(styles.optionsMenuToggle, isDisabled && getModifier(styles, 'disabled'), getModifier(styles, 'plain'), getModifier(styles, 'text'))} >
```

Comment:
nit: it generally makes more sense to use getModifier if a variable prop can be matched to the style object. In this case i would change these to `styles.disabled`, `styles.plain`, etc..

Reply:
Ah I thought that was just how we retrieved modifier styles. I can change it to `styles.modifiers.disabled` and so on.